couchbase/admin/bucket/v1/bucket.proto

```diff
@@ -64,6 +64,9 @@ message ListBucketsResponse {
     optional couchbase.kv.v1.DurabilityLevel minimum_durability_level = 10;
     optional StorageBackend storage_backend = 11;
     ConflictResolutionType conflict_resolution_type = 12;
+    optional bool history_retention_collection_default = 13;
```

Comment:
I think we typically just would call this 'history_retention_enabled', since it's implied that buckets are the 'parent' to a collection, and their options propagate as such?  Any reason for calling it out explicitly and doing it differently here?

Reply:
I think the only reason would be because this is what ns_server calls it.